### PR TITLE
Milestone : final touches :ok_hand: 

### PR DESCRIPTION
**Pull request by ahmedeid6842**
# Add the Finishing Touches to Your Leaderboard List App with Custom Styles. :100: 

improved the application's look and feel by adding a responsive web design for screen sizes up to 768px :computer:, and status messages for successful :white_check_mark: and failed :x: actions. These changes enhance the user experience :+1:, making the application more accessible and user-friendly :smiley:.

### Done :memo:  
- [x] :white_check_mark: Implemented RWD (responsive web design) with `max-width: 768px` to ensure the application fits multiple screen sizes.
- [x] :white_check_mark: Added status messages to provide immediate feedback on whether score additions were successful :heavy_check_mark: or not :x:.

### Highlights :star: :star2: 
- :art: Created a beautiful CSS style for status messages.
- :memo: Wrote a descriptive pull request :writing_hand: that explains the changes made to the codebase.

### Success message :heavy_check_mark: : 

Screenshot from 2023-06-23 02-04-12

### Error message :no_entry_sign: :

Screenshot from 2023-06-23 02-04-29

### RWD  :iphone: 

Screenshot from 2023-06-23 01-57-38

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -28,6 +28,14 @@ <h2>Add you score</h2>
           <input type="number" placeholder="Your score" id="score" required />
           <button type="submit" class="add-btn btn">Submit</button>
         </form>
+        <div class="status">
+          <div class="success hidden">
+            <span>Your score has been added successfully. ✅ 🎯 </span>
+          </div>
+          <div class="error hidden">
+            <span>Something went wrong, please try again. 🚫 </span>
```
Comment (tufoinnkuo10) [inline src/index.html:36]: - [optional] wonderful job adding an error message for wrong submission. Please, I advise you add more styles, even a nice background will make your design more appealing. Nice work.

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @ahmedeid6842 

Your project is complete! Great job for nailing it 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request ✔️ 
- Wireframe design followed✔️ 
- App works as expected✔️ 
- No linter errors✔️ 

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahmedeid6842** [comment]
Thank you so much, @tufoinnkuo10 :pray:, for taking the time and effort to review my code. Your feedback is greatly appreciated :+1:.

**Comment by ahmedeid6842** [comment]
# 🎉🎉🥳🥳

7fb4d55fa09fc4f5426e6d5934d1d02f

**Pull request merged**